sway/desktop/layer_shell.c

```diff
@@ -563,6 +634,9 @@ void handle_layer_shell_surface(struct wl_listener *listener, void *data) {
 	wl_signal_add(&layer_surface->events.unmap, &sway_layer->unmap);
 	sway_layer->new_popup.notify = handle_new_popup;
 	wl_signal_add(&layer_surface->events.new_popup, &sway_layer->new_popup);
+	sway_layer->new_subsurface.notify = handle_new_subsurface;
+	wl_signal_add(&layer_surface->surface->events.new_subsurface,
```

Comment:
We need to remove the `new_subsurface` listener when the layer shell surface is destroyed.

Reply:
Sorry, i forgot about removing the listeners. Fixed it now.